src/rime/dict/dict_compiler.cc

```diff
@@ -236,22 +236,24 @@ bool DictCompiler::BuildTable(int table_index,
     for (const auto& s : collector.syllabary) {
       syllable_to_id[s] = syllable_id++;
     }
-    for (RawDictEntry& r : collector.entries) {
+    for (auto r : collector.entries) {
```

Comment:
nit: copied the reference counted pointer to a temporary variable. `const auto&` avoids the extra reference counting cost.

Reply:
OK, I will fix it soon.